util/rom_chip_info.py

```diff
@@ -57,24 +75,23 @@ def main():
         log.error(
             "Missing ot_version, provide --ot_version or --ot_version_file.")
         raise SystemExit(sys.exc_info()[1])
+    version = int(version, base=16)
 
     outdir = Path(args.outdir)
 
     outdir.mkdir(parents=True, exist_ok=True)
-    out_path = outdir / "chip_info.h"
+    source_out_path = outdir / "chip_info.c"
 
     now = datetime.now()
     wall_time = now.strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
I don't think `now` is the correct method to call here, as it is subject to the current timezone.  `utcnow` would be better.
Even better, we can get this info from bazel as `BUILD_TIMESTAMP` from volatile-status.txt.

In general, I don't think we ever want to encode the timestamp into binaries - that makes them inherently non-reproducible.  However, we _might_ want to make an exception for the ROM, as once we decide on the release version, we make the binary and give it to the manufacturer to build into the chip.

@alphan, do you have an opinion on this?

Reply:
I'd actually prefer not including this in the binary itself. Assuming that we have a reproducible build, a timestamp wouldn't really mean much, right? Someone can build the ROM from the same commit and get the same identical binary any time. We could always treat the committer/author date of the commit hash as the timestamp.

We can also include the build timestamp and potentially other metadata in a hjson or other file during the release if you prefer.

Reply:
Happy to drop the timestamp in the interest of hermeticity. I'm not sure what else to include in chip_info besides the hw_rev and commit hash — anything else is really just a convenience, I think.

Reply:
Good catch on `now` vs `utcnow` though @cfrantz !

Reply:
I'd suggest dropping the timestamp completely.  If we want to put something descriptive (ie: text) into the chip info, I'd suggest some transformation of BUILD_GIT_VERSION from volatile-status.txt.